home.css

```diff
@@ -0,0 +1,473 @@
+  font-weight: bold;
+  color: #585858;
+  font-size: 200%;
+  padding-left: 3%;
+  margin-bottom: 0%;
+  margin-top: 5%;
+}
+
+.venue {
+  position: relative;
+  color: #585858;
+  padding-left: 3%;
+  font-weight: 600;
+  margin-bottom: 5%;
+}
+
+/* program section */
+.program::before {
+  background: url("./pics/x.png") 0, 0 #0a0a0a;
+  content: "";
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  position: absolute;
+  opacity: 0.75;
+}
+
+.program {
+  position: relative;
+}
+
+.main_program {
+  position: relative;
+  color: #fff;
+  padding-top: 10%;
+  padding-bottom: 2%;
+  text-align: center;
+}
+
+.main_program::after {
+  content: '';
+  height: 2px;
+  width: 10%;
+  background: #f35826;
+  position: absolute;
+  bottom: 0;
+  left: 45%;
+}
+
+.activities {
+  padding-top: 5%;
+  list-style-type: none;
+  font-size: 90%;
+  padding-right: 2%;
+  padding-left: 2%;
+}
+
+.activity {
+  display: flex;
+  color: #fff;
+  border: none;
+  background-color: #3c3f46;
+  margin: 2%;
+  margin-left: 2%;
+}
+
+.activities .activity:hover {
+  cursor: pointer;
+  position: relative;
+  border: #fff 2px solid;
+}
+
+.img_activity {
+  position: relative;
+  padding: 3%;
+  width: 50px;
+  height: 50px;
+  margin-top: 3%;
+}
+
+.description {
+  position: relative;
+  display: flex;
+  align-items: center;
+  color: #f35826;
+  font-weight: bold;
+  padding-right: 5%;
+}
+
+.explanation {
+  position: relative;
+  font-size: 92%;
+}
+
+.center_button {
+  position: relative;
+  display: flex;
+  justify-content: center;
+}
+
+.join {
+  padding: 4%;
+  background-color: #f35826;
+  border: none;
+  color: #fff;
+  font-size: 140%;
+  margin: 5%;
+  margin-bottom: 10%;
+}
+
+.join:hover {
+  cursor: pointer;
+}
+
+.join:hover:hover {
+  background-color: #fff;
+  color: #f35826;
+}
+
+.join:hover:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+/* guest section */
+.featured_guests {
+  text-align: center;
+  position: relative;
+  padding-bottom: 2%;
+}
+
+.featured_guests::after {
+  content: '';
+  height: 2px;
+  width: 10%;
+  background: #f35826;
+  position: absolute;
+  bottom: 0;
+  left: 45%;
+}
+
+.guests {
+  list-style-type: none;
+  margin-top: 5%;
+}
+
+.guests li {
+  display: flex;
+}
+
+.guest_img_div {
+  margin-top: 4%;
+}
+
+.guest_img {
+  width: 100px;
+  height: 100px;
+}
+
+.guest_paragraphs {
+  padding-left: 3%;
+}
+
+.name {
+  font-weight: bold;
+}
+
+.work {
+  color: #f35826;
+  font-style: italic;
+}
+
+.work_description {
+  font-size: 95%;
+}
+
+/* desktop version */
+@media (min-width: 481px) and (min-width: 768px) {
+  .hamburger_button {
+    display: none;
+  }
+
+  header::before {
+    top: 11.5%;
+  }
+
+  .warzone_head_logo {
+    width: 100px;
+  }
+
+  .navbar {
+    position: relative;
+    display: flex;
+    justify-content: space-around;
+    -webkit-box-shadow: 0 4px 6px -6px #222;
+    -moz-box-shadow: 0 4px 6px -6px #222;
+    box-shadow: 0 4px 6px -6px #222;
+  }
```

Comment:
- [ ] Great job implementing the navbar. It would be great if all the elements of the list can be aligned horizontally. 😉

Something like this:
![Screen Shot 2021-11-17 at 19 28 30](https://user-images.githubusercontent.com/67211919/142335984-360037a4-672b-4d23-a5d8-d84dc05a24d5.png)

Your navbar:
![Screen Shot 2021-11-17 at 19 45 17](https://user-images.githubusercontent.com/67211919/142336147-91248624-9ac8-4c0c-a0d1-36fa3a48446a.png)

Reply:
Thank you for your awesome observation. I have just fixed that now.